code/modules/research/designs/misc_designs.dm

```diff
@@ -97,6 +97,16 @@
 	category = list("Equipment")
 	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE | DEPARTMENTAL_FLAG_ENGINEERING | DEPARTMENTAL_FLAG_SECURITY | DEPARTMENTAL_FLAG_CARGO
 
+/datum/design/rolling_table
+	name = "Table-with-Wheels"

```

Comment:
Can we call this design(not the structure itself), "Rolly poly rolling table"? Every stupid kickstarter / TV-shop product needs a stupid name.